.github/ISSUE_TEMPLATE/usage_question.md

```diff
@@ -0,0 +1,17 @@
+---
+name: Usage question
+about: If you have a usage question
+title: ''
+labels: question
+assignees: ''
+
+---
+
+** If your issue is a usage question, submit it here instead:**
+- **StackOverflow with the scikit-learn tag: https://stackoverflow.com/questions/tagged/scikit-learn**
+- **Mailing List: https://mail.python.org/mailman/listinfo/scikit-learn**
+- **Gitter: https://gitter.im/scikit-learn/scikit-learn**
+- **For more information, see User Questions: http://scikit-learn.org/stable/support.html#user-question**
+
+We are going to automatically close this issue if this is not link to a bug or
```

Comment:
I don't understand this. As a user, am I allowed to submit a question here or not? Which questions are accepted?

Reply:
Basically I wanted to check if we could directly redirect users to a proper channel (SO or mailing list) instead of getting a bug fix report. But I am happy removing it or getting an alternative.

Reply:
I'm happy to keep this, but maybe say:

```
The issue tracker is used only to report issues and feature requests. For questions, please
use either of the above platforms. Most question issues are closed w/o an answer on this
issue tracker. Thanks for your understanding.
```